packages/data-service/package.json

```diff
@@ -43,15 +43,12 @@
     "bootstrap": "npm run compile",
     "prepublishOnly": "npm run compile",
     "pretest": "npm run compile && mongodb-runner start --port=27018",
-    "test": "cross-env TS_NODE_FILES=true nyc mocha --colors --timeout 15000 -r ts-node/register src/**/*.spec.*",
+    "test": "cross-env TS_NODE_FILES=true nyc mocha",
```

Comment:
was fighting with this a lot, can we have a different task for covered tests (`test-cov`)? And a `test-watch` with `--watch`, maybe only me, but personally I use that a ton and run tests each 3 seconds, i would love to have a way for that to be really fast.